nexus/api/settings/config.json

```diff
@@ -0,0 +1,517 @@
+{
```

Comment:
TODO:
 - add missing settings 
 -  extend settings types
 - add script to generate config to wandb/wandb